src/fconvert/fconvert_tseries.jl

```diff
@@ -377,8 +377,8 @@ _fconvert_lower(F_to::Type{<:Union{Monthly,Quarterly{N},Quarterly,HalfYearly,Hal
 _fconvert_lower(F_to::Type{<:Union{Monthly,Quarterly{N},Quarterly,HalfYearly,HalfYearly{N},Yearly,Yearly{N},Weekly,Weekly{N}}}, t::TSeries{<:Union{BDaily,Daily,<:Weekly}}, method::Val{:max}; ref=:end, skip_all_nans::Bool=false, skip_holidays::Bool=false, holidays_map::Union{Nothing, TSeries{BDaily}}=nothing) where {N} = _fconvert_lower(F_to, t, maximum; ref=ref, skip_all_nans=skip_all_nans, skip_holidays=skip_holidays, holidays_map=holidays_map)
 _fconvert_lower(F_to::Type{<:Union{Monthly,Quarterly{N},Quarterly,HalfYearly,HalfYearly{N},Yearly,Yearly{N},Weekly,Weekly{N}}}, t::TSeries{<:Union{BDaily,Daily,<:Weekly}}, method::Val{:point}; ref=:end, skip_all_nans::Bool=false, skip_holidays::Bool=false, holidays_map::Union{Nothing, TSeries{BDaily}}=nothing) where {N} = _fconvert_lower(F_to, t, method, Val(ref), skip_all_nans=skip_all_nans, skip_holidays=skip_holidays, holidays_map=holidays_map)
-_fconvert_lower(F_to::Type{<:Union{Monthly,Quarterly{N},Quarterly,HalfYearly,HalfYearly{N},Yearly,Yearly{N},Weekly,Weekly{N}}}, t::TSeries{<:Union{BDaily,Daily,<:Weekly}}, method::Val{:begin}; ref=:end, skip_all_nans::Bool=false, skip_holidays::Bool=false, holidays_map::Union{Nothing, TSeries{BDaily}}=nothing) where {N} = _fconvert_lower(F_to, t, Val(:point), Val(ref), skip_all_nans=skip_all_nans, skip_holidays=skip_holidays, holidays_map=holidays_map)
-_fconvert_lower(F_to::Type{<:Union{Monthly,Quarterly{N},Quarterly,HalfYearly,HalfYearly{N},Yearly,Yearly{N},Weekly,Weekly{N}}}, t::TSeries{<:Union{BDaily,Daily,<:Weekly}}, method::Val{:end}; ref=:end, skip_all_nans::Bool=false, skip_holidays::Bool=false, holidays_map::Union{Nothing, TSeries{BDaily}}=nothing) where {N} = _fconvert_lower(F_to, t, Val(:point), Val(ref), skip_all_nans=skip_all_nans, skip_holidays=skip_holidays, holidays_map=holidays_map)
+_fconvert_lower(F_to::Type{<:Union{Monthly,Quarterly{N},Quarterly,HalfYearly,HalfYearly{N},Yearly,Yearly{N},Weekly,Weekly{N}}}, t::TSeries{<:Union{BDaily,Daily,<:Weekly}}, method::Val{:begin}; ref=:end, skip_all_nans::Bool=false, skip_holidays::Bool=false, holidays_map::Union{Nothing, TSeries{BDaily}}=nothing) where {N} = _fconvert_lower(F_to, t, Val(:point), Val(:begin), skip_all_nans=skip_all_nans, skip_holidays=skip_holidays, holidays_map=holidays_map)
+_fconvert_lower(F_to::Type{<:Union{Monthly,Quarterly{N},Quarterly,HalfYearly,HalfYearly{N},Yearly,Yearly{N},Weekly,Weekly{N}}}, t::TSeries{<:Union{BDaily,Daily,<:Weekly}}, method::Val{:end}; ref=:end, skip_all_nans::Bool=false, skip_holidays::Bool=false, holidays_map::Union{Nothing, TSeries{BDaily}}=nothing) where {N} = _fconvert_lower(F_to, t, Val(:point), Val(:end), skip_all_nans=skip_all_nans, skip_holidays=skip_holidays, holidays_map=holidays_map)
```

Comment:
It's interesting here that you cannot re-use `ref`... just a thought.

Reply:
We can't re-use ref because the default ref (`:end`) might conflict with the method, in this case.